test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/WipeTokenAccountPrecompileSuite.java

```diff
@@ -0,0 +1,160 @@
+                                                        .bytecode(WIPE_CONTRACT),
+                                                contractCall(
+                                                                WIPE_CONTRACT,
+                                                                "wipeFungibleToken",
+                                                                asAddress(vanillaTokenID.get()),
+                                                                asAddress(accountID.get()),
+                                                                10L)
+                                                        .payingWith(ACCOUNT)
+                                                        .via("wipeFungibleTxn")
+                                                        .gas(GAS_TO_OFFER))))
+                .then(
+                        getTokenInfo(VANILLA_TOKEN).hasTotalSupply(990),
+                        getAccountBalance(ACCOUNT).hasTokenBalance(VANILLA_TOKEN, 490));
+    }
+
+    private HapiApiSpec wipeNonFungibleTokenHappyPath() {
+        final AtomicReference<AccountID> accountID = new AtomicReference<>();
+        final AtomicReference<TokenID> vanillaTokenID = new AtomicReference<>();
+
+        return defaultHapiSpec("WipeNonFungibleTokenHappyPath")
+                .given(
+                        newKeyNamed(WIPE_KEY),
+                        newKeyNamed(MULTI_KEY),
+                        cryptoCreate(ACCOUNT).exposingCreatedIdTo(accountID::set),
+                        cryptoCreate(TOKEN_TREASURY),
+                        tokenCreate(VANILLA_TOKEN)
+                                .tokenType(NON_FUNGIBLE_UNIQUE)
+                                .treasury(TOKEN_TREASURY)
+                                .wipeKey(WIPE_KEY)
+                                .supplyKey(MULTI_KEY)
+                                .initialSupply(0)
+                                .exposingCreatedIdTo(id -> vanillaTokenID.set(asToken(id))),
+                        mintToken(VANILLA_TOKEN, List.of(copyFromUtf8("First!"))),
+                        mintToken(VANILLA_TOKEN, List.of(copyFromUtf8("Second!"))),
+                        uploadInitCode(WIPE_CONTRACT),
+                        contractCreate(WIPE_CONTRACT),
+                        tokenAssociate(ACCOUNT, VANILLA_TOKEN),
+                        cryptoTransfer(
+                                movingUnique(VANILLA_TOKEN, 1L, 2L)
+                                        .between(TOKEN_TREASURY, ACCOUNT)))
+                .when(
+                        withOpContext(
+                                (spec, opLog) -> {
+                                    final var serialNumbers = new ArrayList<>();
+                                    serialNumbers.add(1L);
+                                    allRunFor(
+                                            spec,
+                                            contractCreate(WIPE_CONTRACT).bytecode(WIPE_CONTRACT),
```

Comment:
It seems you create WIPE_CONTRACT twice. Also bytecode is not needed anymore to be passed explicitly.